src/containers/ProfileDetails/components/RF/index.js

```diff
@@ -0,0 +1,479 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Form, Input, Select, Tooltip  } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import styles from '../index.module.scss';
+
+const { Item  } = Form;
+const { Option } = Select;
+
+const RFForm = ({ details}) => {
+
+    const defaultOptions = (
+        <Select className={styles.Field}>
+          <Option value="enabled">enabled</Option>
+          <Option value="disabled">disabled</Option>
+        </Select>
+      );
+    
+      const setInitialValue = (obj = {}, dataIndex, key, options = {}) => {
+        const val = options.value ? obj[key][options.value][dataIndex] : obj[key][dataIndex];
+        if (val === undefined || val === null) {
+          return 'disabled';
+        }
+        if (val === true) {
+          return 'enabled';
+        }
+        if (val === false) {
+          return 'disabled';
+        }
+    
+        return val;
+      };
+
+    const renderItem = (label, obj = {}, dataIndex, renderInput, options = {}) => (
```

Comment:
remove the param `obj` it is the same everytime. You just need to loop through the radios and call renderInput for each radio